javafx/javafx2.editor/src/org/netbeans/modules/javafx2/editor/FXMLCompletion2.java

```diff
@@ -161,6 +161,8 @@ public void run(ResultIterator resultIterator) throws Exception {
                 }
                 Parser.Result result = resultIterator.getParserResult();
                 ci = CompilationInfo.get(result);
+                // following populates module info, partial fix for [NETBEANS-4832]
+                ci.getElements().getPackageElement("java.lang");
```

Comment:
I understand the trigger, but couldn't it be fixed right in the getPackagElement(), @jlahoda ?

Reply:
Did you mean `getElements()` rather than `getPackageElement()`?

Reply:
I think the expected way would be to get a CompilationController (instead of just CompilationInfo) and call toPhase(ELEMENTS_RESOLVED), or alike, on it.

Reply:
@jlahoda Use CompilationController as suggested. Works, thanks. Please review.